rfcs/README.md

```diff
@@ -1,53 +1,8 @@
-# RFCs
+# Request for Comments (RFCs)
```

Comment:
Should we also move the completed RFCs to a "done" directory? Otherwise, I can envision this directory size increasing to a size where it is difficult to scan its contents.

Reply:
This might be a good idea, but I'm content with waiting until it's a problem to deal with it. I like the idea of them all being centralized, for now.